<?xml version="1.0"?>
<launch>

  <arg name="model" default="$(find reasoner)/owl/ux1.owl"/>
  <arg name="tomasys" default="$(find reasoner)/owl/tomasys.owl"/>
  <arg name="desired_configuration" default="fd_heave_all"/>
  <arg name="reconfiguration_path" default="$(find reasoner)/reconfiguration/"/>

  <node name="reasoner" pkg="reasoner" type="reasoner_node.py" output="screen">
    <param name="model_file" type="string" value="$(arg model)"/>
    <param name="tomasys_file" type="string" value="$(arg tomasys)"/>
    <param name="desired_configuration" type="string" value="$(arg desired_configuration)"/>
    <param name="reconfiguration_path" type="string" value="$(arg reconfiguration_path)"/>
  </node>

</launch>
